### PR TITLE
feat(resolver): Allow CIDR fallback value

### PR DESCRIPTION
**Pull request by erezrokah**
🎉 Thank you for making CloudQuery awesome by submitting a PR 🎉

#### Summary

Related to https://github.com/cloudquery/cq-provider-gcp/issues/255.
The GCP API can return a valid CIDR or an empty string. This change to the SDK allows providers to have a fallback value if the parsing of the CIDR fails.

>I can add this to a bunch of other places too, but didn't do it since didn't find an actual use case for it

---

Use the following steps to ensure your PR is ready to be reviewed

- [x] Read the [contribution guidelines](../blob/main/.github/CONTRIBUTING.md) 🧑‍🎓
- [x] Run `go fmt` to format your code 🖊
- [x] Lint your changes via `golangci-lint run --new-from-rev main` 🚨 (install golangci-lint [here](https://golangci-lint.run/usage/install/#local-installation))
- [x] Update or add tests 🧪
- [x] Ensure the status checks below are successful ✅

### CODE REVIEW

File: provider/schema/resolvers.go [hunk 1]
```diff
@@ -12,6 +12,13 @@ import (
 	"github.com/thoas/go-funk"
 )
 
+type Fallback struct {
+	Value interface{}
```
Comment (erezrokah) [inline provider/schema/resolvers.go:16]: We use a separate type so we can set a `nil` fallback value

**Comment by roneli** [comment]
Why not use columns Default value, the error will still exist. Moreover, I think this is a GCP specific thing, not sure if I like adding this complexity to resolvers

**Comment by erezrokah** [comment]
Maybe the correct way to do it is to add it to `IgnoreError`?

**Comment by erezrokah** [comment]
Ah, I think we already have a way to handle this case. Let me close and see if I can handle in `gcp` provider

**Pull request closed without merging**